packages/next-typescript/package.json

```diff
@@ -5,7 +5,7 @@
   "license": "MIT",
   "repository": "zeit/next-plugins",
   "dependencies": {
-    "@babel/preset-typescript": "7.0.0-beta.42"
+    "@babel/preset-typescript": "7.0.0-rc.1"
```

Comment:
Given that this is an RC, can the dependency be declared as `"^7.0.0-rc.1"` (with `^`)?